profiles/exceptions.py

```diff
@@ -1,6 +1,3 @@
-from profiles.models import Client
```

Comment:
Ran into a cyclical import. Changing `models.py` to not create `SQLAlchemy` looked like it would resolve it, but can't find out how to add the models (https://stackoverflow.com/questions/28789063/associate-external-class-model-with-flask-sqlalchemy).

Reply:
Would a separate `repository.py` module help? Seems strange to have Entity and Repository interface and Repository implementation all in the same place. `exceptions` and `repository` could depend on `models`.
Moreover, `Client` doesn't look like part of the domain but only a configuration class (but don't know what depends on it)

Reply:
Resolved